src/hooks/useChats.ts

```diff
@@ -0,0 +1,23 @@
+import { useFirestoreConnect } from "react-redux-firebase";
+
+import { venueChatsSelector } from "utils/selectors";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useSelector } from "hooks/useSelector";
+
+export const useVenueChatConnect = () => {
+  const venueId = useVenueId();
```

Comment:
IMO we shouldn't be calling this sort of thing 'inside' a hook like this, it feels like it couples it too tightly. My preference in the past has been to pass the `venueId` we want to 'connect' to as a prop. Which will make it easier to write nice isolated tests/etc as well.
```suggestion
import { useSelector } from "hooks/useSelector";

export const useVenueChatConnect = (venueId: string) => {
```

If you make this change though (maybe even if you don't) then we'll need to ensure that the `storeAs` `venueChats` always refers to the same venue. It might be better to make the `storeAs` include the `venueId`.. eg.

```
storeAs: `${venueId}Chats`,
```

(and if we do that, we also need to make sure to update the `useVenueChats` below to correctly reference this as well)

Reply:
Partially fixed in 354167a550c42e93a27cc5c1e431145ff1caeef7, but that implementation doesn't account for the `venueId` changing and how that will impact the  `storeAs` name as highlighted in my above comment.

Reply:
Backlogged the above in https://github.com/sparkletown/internal-sparkle-issues/issues/110